content/posts/addons-by-default.md

```diff
@@ -0,0 +1,100 @@
+title: Read the Docs Addons enabled by default
```

Comment:
I feel like we need to lead with the build data deprecation, not Addons?

Reply:
Hrm, I structured the post in this way because I'd assume that most users won't know what "build data deprecation" means and if they are using it or not.

I think the news "Addons enabled by default" and "What does it means?" and "How it affect my projects?" is a better way to structure it. That's why I went this way.

Reply:
Also, I think that most of the users won't be directly affected by this (meaning, that it won't break their builds). I'm expecting a handful of users using the Sphinx context in a way that can hard break their docs and will require using the extension and/or contacting us to understand what happened.

Reply:
In this context, the _news_ is that "Addons are being enabled by default" and it will have some implicates to the build process; not the other way around.